versions.gradle

```diff
@@ -185,6 +185,7 @@ ext.deps = deps
 def build_versions = [:]
 build_versions.min_sdk = 24
 build_versions.min_sdk_wave = 25
+build_versions.compile_sdk = 31
```

Comment:
I wonder, is this required by any of the updated dependencies?

Reply:
Yes, it is required by `androidx.appcompat:appcompat`:

- version **1.4.2** requires **31**
- version **1.5.1** requires **32** 
- version **1.6.1** (the latest) requires **33**

Reply:
We will use `androidx.appcompat:appcompat` **1.4.2** and `compileSdkVersion` **31**.

Reply:
Nit: this is a bit stupid but mind moving the compilesdk *after* the target ones? I think it visually makes explicity that mindsdk<=targetsdk<=compilesdk